rules/internal/utils/casing.go

```diff
@@ -14,54 +14,27 @@
 
 package utils
 
-// ToUpperCamelCase returns the UpperCamelCase of a string, including removing
-// delimiters (_,-,., ) and using them to denote a new word.
-func ToUpperCamelCase(s string) string {
-	return toCamelCase(s, true, false)
-}
-
-// ToLowerCamelCase returns the lowerCamelCase of a string, including removing
-// delimiters (_,-,., ) and using them to denote a new word.
-func ToLowerCamelCase(s string) string {
-	return toCamelCase(s, false, true)
-}
-
-func toCamelCase(s string, makeNextUpper bool, makeNextLower bool) string {
+// ToKebobCase returns the kebob-case of a word (book-edition).
+func ToKebobCase(s string) string {
```

Comment:
Not a big deal, but would be good to upstream this and other casing to aep-lib-go: https://github.com/aep-dev/aep-lib-go/blob/main/pkg/cases/cases.go

Reply:
I can do that in a future PR.